va/http_test.go

```diff
@@ -517,6 +517,31 @@ func httpTestSrv(t *testing.T) *httptest.Server {
 			http.StatusMovedPermanently)
 	})
 
+	mux.HandleFunc(fmt.Sprintf("/max-redirect"),
+		func(resp http.ResponseWriter, req *http.Request) {
+			http.Redirect(
+				resp,
+				req,
+				fmt.Sprintf("http://example.com:%d/1/max-redirect", httpPort),
+				http.StatusMovedPermanently,
+			)
+		})
+	// A path that sequentually redirects, creating an incrementing redirect
+	// that will terminate when the redirect limit is reached and ensures each
+	// URL is different than the last.
+	for i := 1; i <= maxRedirect+1; i++ {
+		s, x := strconv.Itoa(i), strconv.Itoa(i+1)
+		mux.HandleFunc(fmt.Sprintf("/%s/max-redirect", s),
```

Comment:
nit: I'd make this `/max-redirect/number` rather than `/number/max-redirect`.

Reply:
Updated to use /max-redirect/number